contrib/twij

```diff
@@ -59,25 +59,39 @@
 #   -t TIMEOUT: Total timeout to pass along through curlicue to curl
 #               (curl's --max-time option). Defaults to 60.
 #
+#   -d FALLBACK: Minimum time to sleep in case of an error processing
+#                rate limit status from an earlier response.
+#
+#   -T LONGEST: Maxiumum time to sleep due to rate limiting. Defaults to
+#               900, the length of a Twitter rate-limit window.
+#
 #   -x: Abort (with exit code 3) if a request would be rate-limited.
 #
+#   -X: Abort (with exit code 4) if a delay would be greater than the
+#       maximum specified by -T.
+#
 # Copyright © 2015 Decklin Foster <decklin@red-bean.com>; distributed
 # under the same license as Curlicue.
 
 api_root='https://api.twitter.com/1.1'
 test -n "$TWIJ_CREDS" && creds="$TWIJ_CREDS"
 method=GET
 time_limit=60
+delay_fallback=30
+delay_limit=900
 
-while getopts 'c:f:m:ps:t:vx' opt; do
+while getopts 'c:d:f:m:ps:t:T:vw:xX' opt; do
```

Comment:
Should `w:` be in the parameter list pattern?  I don't see it in the docs above or the `case` statement below.